src/Workspaces/Core/Portable/Remote/WellKnownSynchronizationKind.cs

```diff
@@ -2,16 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
 namespace Microsoft.CodeAnalysis.Serialization
 {
-    // TODO: Kind might not actually needed. see whether we can get rid of this
     internal enum WellKnownSynchronizationKind
     {
-        Null,
```

Comment:
removing this concept entirely.  it was not allowed, and we had so many confusing checks everywhere around it.